src/content/whats-new/2022/03/whats-new-03-29-netlify-logdrain.md

```diff
@@ -0,0 +1,28 @@
+---
+title: ‘View Netlify logs data with New Relic'
+summary: 'Add observability to your Jamstack application with the New Relic Netlify plugin and quickstart'
+releaseDate: '2022-03-30'
+learnMoreLink: ' 
+getStartedLink: '
+---
+Built for enterprise customers who want visibility of log data without the manual work. New Relics integration provides developers to import logs into the log management solution of their choice for troubleshooting and performance monitoring, or maintain them for historical purposes. Learn more here.
+
+Doing so gives you the following benefits:
+
+* User traffic monitoring: Identify traffic from bots and bad actors, which can help your security and compliance engineers proactively defend your sites.
+* Access and analyze your Netlify traffic logs: Traffic logs are only accessible through log drains, not the Netlify UI.
+* Long-term log retention: The Netlify console provides a 24-hour storage window for logs. The Netlify console provides a 24-hour storage window for function logs. With New Relic One, you get long-term cold storage for your Netlify log data and the ability to make month-over-month comparisons.
+* User agent analytics: Analyze your user agents for performance issues.
+* Session tracking: Gain insight into user activity and journeys across your sites.
+
+## How to configure pluging with New Relic
```

Comment:
@maicadk, should "pluging" be "plug-in"?

Reply:
Yes. Plug-in

Reply:
@maicadk, OK, I got it!